packages/core-mobile/package.json

```diff
@@ -102,6 +102,11 @@
     "es6-promise": "4.2.8",
     "ethers": "6.8.1",
     "events": "3.3.0",
+    "expo": "50.0.21",
+    "expo-constants": "15.4.6",
+    "expo-linking": "6.2.2",
+    "expo-router": "3.4.10",
+    "expo-status-bar": "1.11.1",
```

Comment:
are we using expo-status-bar anywhere?

Reply:
not directly, but expo-router depends on it(as a peerDependency) so we should have it.

https://docs.expo.dev/router/installation/#install-dependencies